parsers/vim.c

```diff
@@ -594,11 +596,14 @@ static bool parseVimLine (const unsigned char *line, int infunction)
 		parseAutogroup (skipWord (line));
 	}
 
-	else if (wordMatchLen (line, "let", 3))
+	else if (wordMatchLen (line, "let", 3) || wordMatchLen (line, "const", 4))
```

Comment:
Mathcing "const" is not needed here.

Reply:
You are right. I forgot to revert my old patch, sorry.